src/lib/components/Text/Text.module.scss

```diff
@@ -17,12 +17,16 @@
   font-weight: 450;
 
   &:hover {
-    background-color: #5BC8DC;
+    background-color: #5bc8dc;
   }
 }
 
 .active {
-  background-color: rgb(246, 217, 24);
+  background-color: #f6d918;
+}
+
+.highlighted {
+  background-color: #b8f5b9;
```

Comment:
This is something I'd really like feedback on. I'm not very good with colors. Does this look like a good highlighting color? Would there be a better color to use? The only requirement is that it be different from active (`#f6d918`) and hover (`#5bc8dc`).

Reply:
I left my feedback